src/dataSources/postgres/records.ts

```diff
@@ -0,0 +1,21 @@
+import { FileMetadata } from '../../models/File';
+
+export interface FileRecord {
```

Comment:
This interface is the same as in useroffice-core, it could be an idea to do some sharing like we do between frontend and backend

Reply:
Yes, sharing types is still we have to figure out how to do. In the core you have the graphql types which are generated for FE, but I'm still not sure how to do that between backends as there are some lower level types, which aren't exposed in graphql.